2021-04-21 16:53:23,045 INFO    MainThread:16028 [wandb_setup.py:_flush():69] setting env: {}
2021-04-21 16:53:23,045 INFO    MainThread:16028 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-21 16:53:23,046 INFO    MainThread:16028 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210421_165323-ypd2mfzj/logs/debug.log
2021-04-21 16:53:23,046 INFO    MainThread:16028 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210421_165323-ypd2mfzj/logs/debug-internal.log
2021-04-21 16:53:23,046 INFO    MainThread:16028 [wandb_init.py:init():369] calling init triggers
2021-04-21 16:53:23,046 INFO    MainThread:16028 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-21 16:53:23,046 INFO    MainThread:16028 [wandb_init.py:init():418] starting backend
2021-04-21 16:53:23,046 INFO    MainThread:16028 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-21 16:53:23,057 INFO    MainThread:16028 [backend.py:ensure_launched():123] starting backend process...
2021-04-21 16:53:23,062 INFO    MainThread:16028 [backend.py:ensure_launched():128] started backend process with pid: 16053
2021-04-21 16:53:23,066 INFO    MainThread:16028 [wandb_init.py:init():423] backend started and connected
2021-04-21 16:53:23,067 INFO    MainThread:16028 [wandb_init.py:init():465] updated telemetry
2021-04-21 16:53:23,068 INFO    MainThread:16028 [wandb_init.py:init():484] communicating current version
2021-04-21 16:53:24,311 INFO    MainThread:16028 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-21 16:53:24,311 INFO    MainThread:16028 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-21 16:53:24,530 INFO    MainThread:16028 [wandb_init.py:init():522] starting run threads in backend
2021-04-21 16:53:24,556 INFO    MainThread:16028 [wandb_run.py:_console_start():1524] atexit reg
2021-04-21 16:53:24,557 INFO    MainThread:16028 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-21 16:53:24,557 INFO    MainThread:16028 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-21 16:53:24,559 INFO    MainThread:16028 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-21 16:53:24,559 INFO    MainThread:16028 [wandb_init.py:init():546] run started, returning control to user process
2021-04-21 16:53:36,047 INFO    MainThread:16028 [wandb_run.py:_config_callback():749] config_cb None None {'n_estimators': 10000, 'learning_rate': 0.07, 'eval_metric': 'AUC', 'loss_function': 'Logloss', 'random_seed': 42, 'metric_period': 100, 'od_wait': 100, 'depth': 6, 'rsm': 0.8}
2021-04-21 16:53:42,720 INFO    MainThread:16028 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-21 16:53:42,720 INFO    MainThread:16028 [wandb_run.py:_restore():1466] restore
2021-04-21 16:53:44,946 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 569
}

2021-04-21 16:53:45,551 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 5348
}

2021-04-21 16:53:45,920 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:46,022 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:46,124 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:46,226 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:46,328 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:46,430 INFO    MainThread:16028 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5348
  total_bytes: 5348
}

2021-04-21 16:53:47,523 INFO    MainThread:16028 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-21 16:53:47,524 INFO    MainThread:16028 [wandb_run.py:_show_history():1809] rendering history
2021-04-21 16:53:47,524 INFO    MainThread:16028 [wandb_run.py:_show_files():1838] logging synced files
